api/envoy/extensions/filters/http/grpc_json_reverse_transcoder/v3/transcoder.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.grpc_json_reverse_transcoder.v3;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.grpc_json_reverse_transcoder.v3";
+option java_outer_classname = "TranscoderProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/grpc_json_reverse_transcoder/v3;grpc_json_reverse_transcoderv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: gRPC-JSON reverse transcoder]
+// gRPC-JSON reverse transcoder :ref:`configuration overview <config_http_filters_grpc_json_reverse_transcoder>`.
+// [#extension: envoy.filters.http.grpc_json_reverse_transcoder]
+
+// [#next-free-field: 6]
+// GrpcJsonReverseTranscoder is the filter configuration for the gRPC JSON
+// reverse transcoder. The reverse transcoder acts as a bridge between gRPC
+// client and an HTTP/JSON server, converting the gRPC request into HTTP/JSON
+// for the HTTP backend and the HTTP/JSON response back to gRPC for the gRPC
+// client.
+message GrpcJsonReverseTranscoder {
```

Comment:
I know "reverse" here makes sense since there is already a `GrpcJsonTranscoder`, but I'm wondering if a better name would be `GrpcToJsonTranscoder`, making the direction clear.

Reply:
`GrpcToJsonTranscoder` may make it sound bit confusing, as it doesn't spell that different from `GrpcJsonTranscoder` and feels like both the filters are doing the same thing. That's why I think "reverse" makes more sense.

Reply:
> GrpcToJsonTranscoder may make it sound bit confusing, as it doesn't spell that different from GrpcJsonTranscoder 

That's a fair point.